common-main.c

```diff
@@ -86,5 +86,10 @@ int common_exit(const char *file, int line, int code)
 	check_bug_if_BUG();
 	trace2_cmd_exit_fl(file, line, code);
 
+	/**
+	 * Some software (such as security software) may change buffer type of stdout.
+	 * It will no output without fflush manually.
+	 */
+	fflush(stdout);
```

Comment:
Good! Could you move this to the beginning of the function, though?

Reply:
Thanks for your advice. I will change my commit soon.